packages/desktop/views/onboarding/views/network-setup/views/ChooseNetworkView.svelte

```diff
@@ -39,7 +39,7 @@
 
     function onNetworkSelectionClick(networkId: NetworkId): void {
         if (networkId !== NetworkId.Custom) {
-            const network = NETWORK?.[networkId]
+            const network = NETWORK_METADATA_MAP[networkId]
             const clientOptions = getDefaultClientOptions(networkId)
             updateOnboardingProfile({ network, clientOptions })
```

Comment:
infers type as IStardustNetwork which is missing chains?

Reply:
Should there be a getDefaultNetwork function that can build the default network based on the id. And add the default chains. at the moment default chains is [] but that will change soon.